### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by orestis-kaplan**
For code review

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <link rel = "stylesheet"  type = "text/css" href = "style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title></title>
```
Comment (heyalvaro) [inline index.html]: Why is this empty?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            <img src="img/bell-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/message-icon.png" width="40" height="25" style="padding:17px;float:right;" />
+            <img src="img/app-menu-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/camera-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+
+    </header>
+    <div id="margine">
+
+    <div class="youtube_video">
+        <iframe width="100%" height="100%" src="https://www.youtube.com/embed/ZnuwB35GYMY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```
Comment (heyalvaro) [inline index.html]: The width and height properties of the iframe use `px` as units. You only have to assign a digit. If you want to use a different unit try with css

Comment (heyalvaro) [reply index.html]: The frameborder attribute on the iframe element is obsolete. Use CSS instead.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+    padding: 0;
+}
+
+/* Header bar*/
+.youtube_video{
+  position: relative;
+  width: 69%;
+  float: left;
+  height: 470px;
+  /*margin-left: 16%;
```
Comment (heyalvaro) [inline style.css]: Please avoid commenting code on final submissions

File: style.css [hunk 4]
```diff
@@ -0,0 +1,124 @@
...
+    border: 1px solid #D3D3D3;
+    border-radius: 0 2px 2px 0;
+    font-family: Arial, Helvetica, sans-serif;
+    color: #bdc3c7;
+}
+.search-button{
+  height: 30px;
+  width: 70px;
+}
+/* Youtube Video*/
```
Comment (heyalvaro) [inline style.css]: Please avoid commenting code on final submissions

File: index.html [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    <meta charset="utf-8">
+    <link rel = "stylesheet"  type = "text/css" href = "style.css" />
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <title></title>
+  </head>
+  <body>
+    <header>
+            <img src="img/menu_bar.png" width="25" height="25" style="padding:17px;" />
+            <img src="img/youtube.png" width="100" height="64" />
+            <form class="search-form">
```
Comment (heyalvaro) [inline index.html]: If you just want the input you don't need to add the whole form

File: index.html [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+              <input type="search" placeholder="search" >
+              <button type="submit" class="search-button">
+                <img src="img/search-icon.png"/>
+              </button>
+            </form>
+            <img src="img/account-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/bell-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/message-icon.png" width="40" height="25" style="padding:17px;float:right;" />
+            <img src="img/app-menu-icon.png" width="25" height="25" style="padding:17px;float:right;" />
+            <img src="img/camera-icon.png" width="25" height="25" style="padding:17px;float:right;" />
```
Comment (heyalvaro) [inline index.html]: Grouping these items in a `div` seems beneficial, don't you think?

File: index.html [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+
+    </header>
+    <div id="margine">
```
Comment (heyalvaro) [inline index.html]: You are using percentages for the `margin-left` and `margin-right` properties and the result is that if the window width changes, the alignment breaks. Try it out, change the browser width to half the screen and see what happens.

**Comment by dipto0321** [comment]
Hi @userman95  and @efrapp ,
You did a nice job 👍 . Just a couple of things you need to do. Right below 👇 need to fix:

1.  Please update your readme file. Add a short description of your work. Don't need to mention about you are doing Odin or microverse project. After that add the contributor's name with their GitHub profile link. Try to use markdown to give this readme an awesome look. It's easy ( for inspiration you can see other's student readme ). Another thing rather than naming it "HTMLandcss" rename it with a meaningful one.

2. [Fix this problem](https://prnt.sc/m9jplr)

3. When you are using your own custom class try to name it to kebab-case.

Except those ☝️ all are good. The code is clean with indentation good. You can find some good plugins for your text editor to beautify your code. Use those. Try to build a habit of comment ( should be meaningful ) in your code from your future project.

Please use [w3validator](https://validator.w3.org) and [CSS lint](http://csslint.net/) and try to fix all issues.

**Note:** After fixing, I think you can merge it into the master.

**Pull request merged**